src/util/motionModel/motionModelController.ts

```diff
@@ -31,8 +31,16 @@ export async function MotionModelController() {
       await session.checkObjectDetectionService();
     }
 
+    // if sessions last ingested time is 0, query last time
+    let queryTime = session.lastIngestedTime;
+    if (session.lastIngestedTime === 0){
```

Comment:
Let's keep all this logic inside driveSession,
so motionModel just keeps calling `session.getLastTime()` function

Reply:
Done